scripts/ci-env.ps1

```diff
@@ -20,7 +20,7 @@ switch ($name) {
         return "2023.2.20f1"
     }
     "unity6000" {
-        return "6000.0.25f1"
+        return "6000.0.24f1"
```

Comment:
odd, rolling back a version? cc @vaind

Reply:
the original code for the Unity version updater script relied on parsing the web page. It has flaked once before, this is the second time (that I know of) so I've change it in https://github.com/getsentry/sentry-unity/pull/1880